docs/requirements.txt

```diff
@@ -7,3 +7,4 @@ jupytext
 jupyter
 matplotlib
 numpy
+nltk
```

Comment:
Is this needed here?

Reply:
I ask because these dependencies are used to build the docs, so I wouldn't expect nltk to be needed.

Reply:
Removed from here